proposals/20210707-xgboost_evaluator.md

```diff
@@ -0,0 +1,128 @@
+TFX XGBoost Evaluator - Draft Project Proposal
+
+This is the project proposal for the tfx-addons project https://github.com/tensorflow/tfx-addons/issues/13.
+
+# Project Proposal
+
+**Your name:** Daniel Kim
+
+**Your email:** danielk@twitter.com
+
+**Your company/organization:** Twitter
+
+**Project name:** XGBoost Evaluator Component
+# Project Description
+Add support for evaluating XGBoost model in the standard component Evaluator. 
```

Comment:
Can we also support a inherited Evaluator that can be easily used for XGBoost without having to set custom hooks?

Reply:
Sure - that shouldn't be too much of an issue, if needed.